.github/workflows/release.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
         dbmode:
           - 'dbless'
           - 'postgres'
+        enterprise:
+          - 'yes'
```

Comment:
If this matrix entry doesn't add any configuration (e.g. a matrix generally has at least 2 distinct entries), then it can be dropped:

```suggestion
```

You can simply add a step for the two existing db modes as you've done below.

Reply:
The release.yaml is an entrance specifying "enterprise" enablement or not.

Reply:
I understand that this was the intention but what I'm getting at is that implementing this as a matrix value has a nil effect when there is only one element in the matrix (this is effectively a misuse of the matrix configuration).

If the goal is to make sure the enterprise tests run in parallel with the other tests, use a whole new [job](https://docs.github.com/en/actions/learn-github-actions/introduction-to-github-actions#jobs) instead and you'll get the desired effect, and you can add a matrix for `dbmode` there so that gh actions will run enterprise dbless and postgres in parallel as well.

Reply:
Add some concrete data under enterprise : the repo and tags which integration uses to launch the enterprise image. 
Also, follow the idea of create a separate job for enterprise integration tests, which better provide clarity

Reply:
https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR171-R172